rust/kernel/init.rs

```diff
@@ -235,7 +235,7 @@ pub mod macros;
 ///
 /// # Examples
 ///
-/// ```rust
+/// ```
```

Comment:
I know that `rust` is the default, but I still put that there, because it makes it easier to copy paste the example to a different place. Not sure if we have a standard regarding this though...

Reply:
That makes sense. On one hand, it adds some boilerplate; on the other hand, it is convenient when copying it to an standalone Markdown document.

In the `kernel` crate, the only `rust`-using ones are in `init`. In the Rust `library`/`compiler` folders, I see the majority of the code does not use it (an order of magnitude difference or more).

I guess we can poll / ask others for this one, e.g. how often they copy into another document.